s3backgrounddelete/s3backgrounddelete/IEMutil.py

```diff
@@ -49,5 +52,32 @@ def __init__(self, loglevel, eventcode, eventstring):
         self.log_iem()
 
     def log_iem(self):
-        IEM = "IEC:%sS%s:%s" % (self.severity, self.eventCode, self.eventString)
-        syslog.syslog(IEM)
+        self.send(self.module, self.eventCode, self.severity, self.eventString)
+
+    @classmethod
+    def send(self, module, event_id, severity, message_blob):
+        """Send the message."""
+        source = 'S'
+        component= 'S3'
+        EventMessage.init(component, source)
```

Comment:
Is it required for every time ? - if no move to __init()__

Reply:
done